ntc_templates/templates/cisco_ios_show_interface_transceiver.textfsm

```diff
@@ -1,11 +1,19 @@
 Value INTERFACE (\w+\d+\/\S+)
 Value TEMPERATURE (\d+.\d+)
 Value VOLTAGE (\d+.\d+)
+Value CURRENT (\d+.\d+)
 Value TX_PWR (\S+)
 Value RX_PWR (\S+)
 
 Start
-  ^${INTERFACE}\s+${TEMPERATURE}\s+${VOLTAGE}\s+${TX_PWR}\s+${RX_PWR}\s+ -> Record
+  ^${INTERFACE}\s+${TEMPERATURE}\s+${VOLTAGE}\s+${CURRENT}?\s+${TX_PWR}\s+${RX_PWR}\s* -> Record
   # Capture time-stamp if vty line has command time-stamping turned on
   ^Load\s+for\s+
   ^Time\s+source\s+is
+  ^(\s+Optical){2}
+  ^\s+Temperature\s+Voltage(\s+Current)?(\s+[TR]x\s+Power){2}
+  ^Port\s+\(Celsius\)\s+\(Volts\)(\s+\(mA\))?\s+(\(dBm\)\s*){2}
+  ^(If\s+device|\+\+|NA|mA|---+)
+  ^No transceiver present
+  ^Diagnostic Monitoring is not implemented\.
```

Comment:
I see the error messages are in here -- all good.

Swapping regexes in for literal spaces.

```suggestion
  ^No\s+transceiver\s+present
  ^Diagnostic\s+Monitoring\s+is\s+not\s+implemented\.
```

Reply:
That commit message reads pretty badly in hindsight.

_For clarification I swapped literal whitespace for regexes, not the other way around._